qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -64,17 +66,18 @@ def level_1_pass_manager(transpile_config):
     coupling_map = transpile_config.coupling_map
     initial_layout = transpile_config.initial_layout
     seed_transpiler = transpile_config.seed_transpiler
+    backend_properties = getattr(transpile_config, 'backend_properties', None)
 
     # 1. Use trivial layout if no layout given
-    _given_layout = SetLayout(initial_layout)
+    _set_initial_layout = SetLayout(initial_layout)
 
     def _choose_layout_condition(property_set):
         return not property_set['layout']
 
-    _choose_layout = TrivialLayout(coupling_map)
-
     # 2. Use a better layout on densely connected qubits, if circuit needs swaps
-    _layout_check = CheckMap(coupling_map)
+    def _not_perfect_yet(property_set):
+        return property_set['trivial_layout_score'] is not None and \
```

Comment:
`property_set['trivial_layout_score'] is not None` is the guard to make sure we don't write over a user-supplied layout, right?

Reply:
correct.